sdk/src/wallet/account/operations/output_finder.rs

```diff
@@ -154,7 +154,10 @@ impl AccountHandle {
 
             self.sync(sync_options).await?;
 
-            let output_count = self.read().await.unspent_outputs.len();
+            let output_count = {
+                let account = self.read().await;
+                account.unspent_outputs.len()
+            };
```

Comment:
What's that for ?

Reply:
lifetime shortening

Reply:
yeah, it's just that clippy can't see through it ... I changed it back and added an exception as this was perfectly fine Rust.